homeassistant/components/tag/__init__.py

```diff
@@ -118,10 +118,19 @@ async def async_scan_tag(
     if DOMAIN not in hass.config.components:
         raise HomeAssistantError("tag component has not been set up.")
 
+    helper = hass.data[DOMAIN][TAGS]
+
+    # Get name from helper, default value None if not present in data
+    tag_name = None
+    if (tag_data := helper.data.get(tag_id)) is not None:
```

Comment:
```suggestion
    if tag_data := helper.data.get(tag_id):
```

It can be simplified.